### ISSUE TITLE
Display METAR when using SimBrief

### ISSUE THREAD

**Issue by enriqueo62** [flybywiresim/aircraft#5660]
### Aircraft Version

Development

### Description

A few of us are experiencing CTD when using the PRINT feature in the MCDU, and I liked the feature of having the METAR in the EFB when getting ready to setup the approach without having to go to AOC to open the received message. I think it would be great to bring the METAR import feature to the EFB. The current info for the QNH is in hPa, and the wind is fine but the ground temperature is missing.

### Discord Username (optional)

enriqueo62

**Comment by P1NBA11ER** [comment]
I agree. Having the QNH switching to respect the geographic location, or be a setting would be nice and is something I’ve always looked at and wondered aloud about. Also, loading the metar/atis info to the efb would be a welcome addition for sure.

**Comment by ErickSharp** [comment]
I can pick this request up soon :)

**Comment by tracernz** [comment]
We have a good METAR tool on the EFB home page now.
